packages/flutter_tools/lib/src/ios/simulators.dart

```diff
@@ -637,6 +639,8 @@ class _IOSSimulatorLogReader extends DeviceLogReader {
     _systemProcess.stdout.transform(UTF8.decoder).transform(const LineSplitter()).listen(_onSystemLine);
     _systemProcess.stderr.transform(UTF8.decoder).transform(const LineSplitter()).listen(_onSystemLine);
 
+    // Callback hookups' Futures are uninteresting.
+    // ignore: unawaited_futures
     _deviceProcess.exitCode.whenComplete(() {
```

Comment:
@devoncarew maybe you can help us. Is this method intended to end before the simulator is presumably closed?